front-end/src/views/PublishPage/index.vue

```diff
@@ -35,15 +35,20 @@
     },
     methods: {
       async fetchPrefilledPublication() {
-        const prefilledPublication = await fetch(`${this.$api_url}/get-prefilled-publication`, {
+        const response = await fetch(`${this.$api_url}/get-prefilled-publication`, {
           method: "GET",
           headers: {
             'Content-Type': 'application/json',
             'Authorization': 'Bearer ' + this.jwt
           },
         });
 
-        return this.prefilledPublication = await prefilledPublication.json();
+        if (response.status === 401) {
+          localStorage.removeItem('jwt');
+          location.reload();
+        }
```

Comment:
Peut-être on doit avoir un authenticatedGet dans `submit-actions` qui gère ça et get-diagnostics-for-canteen dans KeyMeasures.js

Reply:
C'était prévu mais avec le nouveau système d'authentication et vuex ça va être obsolète.